### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by andrewardito**
### What does this PR do?
Makes the Agent Installation page a bit more user-friendly and consistent with other places we document to steps to set up APM

### CODE REVIEW

File: content/en/tracing/send_traces/_index.md [hunk 1]
```diff
@@ -23,7 +23,9 @@ further_reading:
   text: "ECS Fargate APM setup"
 ---
 
-To use APM, start by sending your [traces][1] to Datadog, and then [configure your environment](#configure-your-environment). You can send traces to Datadog in multiple different ways depending on your system setup: including using the [Datadog Agent locally](#datadog-agent), [on containers](#containers), and [several other ways](#additional-environments). For the full overview of all of the steps to set up APM, see the [APM overview][2].
+Configuring your application to send [traces][1] to Datadog involves two main steps: Configuring the Datadog Agent and [Instrumenting your application][14]. 
+
+Depending on your environment, there are different ways to configure the Datadog Agent. Instructions are provided here for [installing the Datadog Agent locally](#datadog-agent), [on containers](#containers), and [several other ways](#additional-environments).
```
Comment (kayayarai) [inline content/en/tracing/send_traces/_index.md:28]:
```suggestion
Depending on your environment, there are different ways to configure the Datadog Agent. See the instructions for installing the Datadog Agent [locally](#datadog-agent), [on containers](#containers), and [several other ways](#additional-environments).
```

File: content/en/tracing/send_traces/_index.md [hunk 2]
```diff
@@ -79,7 +81,7 @@ See our guide on setting the [`env` tag and an additional primary tag for scopin
 
 ## Next steps
 
-Next, [Instrument your application][14]. For the full overview of all of the steps to set up APM, see the [APM overview][2].
+Next, [Instrument your application][14]. For the full overview of all of features and configurations of APM, see the [APM overview][2].
```
Comment (kayayarai) [inline content/en/tracing/send_traces/_index.md:84]:
```suggestion
Next, [instrument your application][14]. For the full overview of all of features and configurations of APM, see the [APM overview][2].
```

File: content/en/tracing/send_traces/_index.md (hunk 1 above)
Comment (kayayarai) [reply content/en/tracing/send_traces/_index.md:28]: Also, this kinda just skips over serverless. Want to list it here too?

**Review by kayayarai — CHANGES_REQUESTED** [review]
some suggestions for this

File: content/en/tracing/send_traces/_index.md (hunk 1 above)
Comment (andrewardito) [reply content/en/tracing/send_traces/_index.md:28]: Yeah Serverless we have the separate page for serverless that we'd want to direct people to if they are here.  What's a natural way to do that?

**Review by kayayarai — APPROVED** [review]

**Pull request merged**